### ISSUE TITLE
A lots of copilot related keybind items for choose, I can not understand

### ISSUE THREAD

**Issue by lyleLH** [intitni/CopilotForXcode#168]
- [ ] I have checked FAQ, and there is no solution to my problem
- [ ] I have searched the existing issues, and there is no existing issue for my problem 

<img width="942" alt="图片" src="https://user-images.githubusercontent.com/13076897/233829870-243f58c8-9a31-4668-8918-8781fe41b008.png">

**Describe the bug and the expected behavior**
A clear and concise description of what the bug is. And a clear and concise description of what you expected to happen.

**To reproduce**
1.add custom command in copilot app setting page 
2.goto xcode keymap setting 

Xcode Version 14.3 (14E222b)

**Additional information (optional)**
macOS version: 13.3
Copilot for Xcode version: 
Node version: 19.6.1

**Comment by KieranConlon** [comment]
I am getting the same.
As far as I can tell, every time Xcode is restarted, additional entries for "Explain Selection" and "Add Documentation to Selection" are added to the key bindings list.

<img width="942" alt="image" src="https://user-images.githubusercontent.com/33759139/233831282-13da62c1-6e7d-42e5-89be-4f49a0addcd9.png">

**Comment by intitni** [comment]
I have submitted a new release to stop the duplication. But I haven't find a way to remove already created ones.

**Comment by intitni** [comment]
The keybindings are store at `~/Library/Developer/Xcode/UserData/KeyBindings/`. Please open it in a text editor and delete anything that looks like 

```
<dict>
  <key>Action</key>
  <string>_invokeSourceEditorExtensionCommand:</string>
  <key>Alternate</key>
  <string>NO</string>
  <key>CommandID</key>
  <string>IDESourceEditorExtension:dev.com.intii.CopilotForXcode.EditorExtension,IDESourceEditorExtensionCommand:dev.com.intii.CopilotForXcode.EditorExtensionCustomCommand3052176549756101770</string>
  <key>Group</key>
  <string>Editor Menu for Source Code</string>
  <key>GroupID</key>
  <string>Xcode.IDEPegasusSourceEditor.MenuDefinition.Editor</string>
  <key>GroupedAlternate</key>
  <string>NO</string>
  <key>Navigation</key>
  <string>NO</string>
  <key>Parent Title</key>
  <string>Copilot Dev</string>
  <key>Title</key>
  <string>Explain Selection</string>
</dict>
```

**Comment by lyleLH** [comment]
you guys are blazingly fast

**Comment by KieranConlon** [comment]
Just a quick update on removing the extra 'rouge' keybinding entries.
As I had already created key bindings for the new options, I did try to selectively remove the entries where `CommandID` contained `EditorExtensionCustomCommand` that also did not have any key binding setup. 
These are entries without...
``` XML
<key>Keyboard Shortcut</key>
<string>[your-key-binding]</string>
```

However, they still got re-created the next time I started Xcode.

I then deleted all entries where `CommandID` contains `EditorExtensionCustomCommand`, restarted Xcode, and it worked correctly - obviously, I just had to re-assign the key bindings again.

Anyway, hope this is helpful.
